challenge.md

```diff
@@ -32,19 +32,19 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
- 
+ Relative; that path could exist potentially multiple times out in the world. It means one thing to this computer, but other things on other computers.
```

Comment:
Nope, this is actually an Absolute path. The reason it's Absolute is that no matter where we might be in the filesystem - `/`, `~`, `/Users` - the link above will _always_ direct us to the same place: the desktop. As a rule of thumb, paths that start with `/` are all Absolute paths.